pkgs/applications/networking/instant-messengers/qtox/default.nix

```
@@ -52,17 +49,16 @@ mkDerivation rec {
     openal
     pcre
     qrencode
+    qt6.qtbase
+    qt6.qtsvg
     sqlcipher
   ] ++ lib.optionals stdenv.hostPlatform.isDarwin [ AVFoundation ];
```

```suggestion
  ];
```
Very likely not needed.